### ISSUE TITLE
TodoMVC has broken build order

### ISSUE THREAD

**Issue by dws-dave** [dwstech/muster-examples#2]
The production build of the TodoMVC app crashes on startup with the `hasher type` error.

```
Uncaught Error: Unable to type create hasher: unrecognised type
    at We (index.js:10)
    at e.map.e (index.js:10)
    at Array.map (<anonymous>)
    at e (index.js:10)
    at Object.assign.e (index.js:10)
    at We (index.js:10)
    at e (index.js:10)
    at Object.assign.e (index.js:10)
    at We (index.js:10)
    at Object.keys.map.t (index.js:10)
```

Can be seen in action at [Netlify](https://cranky-lamarr-2c3399.netlify.com/).

App works fine in dev mode, likely the import order is wrong resulting in over-aggressive tree-shaking.

**Comment by dws-dave** [comment]
Diagnosed as being caused by the imports [here](https://github.com/dwstech/muster-examples/blob/master/todo-mvc/src/app.js#L2) which seem to be breaking tree-shaking. Moving the `saveItemsToLocalStorage` function to a standalone file and importing it fixes the problem.

**Comment by dws-dave** [comment]
Closed via #4
